src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -24,7 +24,7 @@ export default class ArrayField extends React.Component {
     super(props);
 
     // Throw an error if there’s no viewField (should be React component)
-    if (typeof this.props.uiSchema['ui:options'].viewField !== 'function') {
+    if (!isReactComponent(this.props.uiSchema['ui:options'].viewField)) {
```

Comment:
Oh, I just noticed that line 13 & 16 in this file both come from the same `utilities/ui` file. Any chance you can include that in this PR? 🥺

Reply:
Technically, they're different files. But I believe they're duplicate files. Do you still want me to merge the lines?

Reply:
naw, I thought they were the same file.